apps/extension/src/final/extension-popup/components/trading-copilot-dialogs/trading-copilot-dialogs.tsx

```diff
@@ -119,9 +119,9 @@ const TradingCopilotDialog = ({
           <div className="grid grid-cols-[48px,1fr] gap-2">
             <LazyImage
               src={avatarQuery.data}
-              className="size-12 rounded-full border border-neutral-400 bg-neutral-200"
+              className="size-12 rounded-full border border-neutral-300 bg-neutral-200"
```

Comment:
Nice catch with the border for the fallback person icon, but it's different for the avatar:
![image](https://github.com/user-attachments/assets/3f144829-5216-4011-867f-cef87da5db04)
![image](https://github.com/user-attachments/assets/f29505b7-69a4-4563-8fac-d6f1c88daf96)